python/lsst/meas/astrom/matchOptimisticB/matchOptimisticBContinued.py

```diff
@@ -367,9 +375,9 @@ def _doMatch(self, refCat, sourceCat, wcs, refFluxField, numUsableSources, minMa
         @param[in] numUsableSources  number of usable sources (sources with known centroid
             that are not near the edge, but may be saturated)
         @param[in] minMatchedPairs  minimum number of matches
-        @param[in] maxMatchDist  maximum on-sky distance between reference objects and sources
-            (an lsst.afw.geom.Angle); if specified then the smaller of config.maxMatchDistArcSec or
-            maxMatchDist is used; if None then config.maxMatchDistArcSec is used
+        @param[in] toleranceStruct lsst.pipe.base.Struct containing
```

Comment:
It's a habit of mine to name the ending of a variable it's type. Certainly if we decide to go with the MatchObject or MatchTolerance class we can do this but as is I like the type to be in the name.